galaxy/values.yaml

```diff
@@ -180,7 +180,7 @@ postgresql:
   existingDatabase:
   galaxyDatabaseUser: galaxydbuser
   # Password is autogenerated and stored in a secret if not specified
-  # galaxyDatabasePassword:
+  galaxyDatabasePassword: galaxydbpassword
```

Comment:
We shouldn't hardcode the password. It's convenient for dev but not a good default overall